src/Loading/Loading.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { ReactComponent as SpinnerLogo } from './assets/tri_logo_monochrome.svg';
+import { StyledLoadingContainer } from './loadingStyles';
+import { ThemeProvider } from '@emotion/react';
+import theme from '../styles/theme';
+export interface LoadingProps {
+    /**
+     * Determines if the component should rotate.
+     */
+    animated?: boolean;
+    /**
+     * The direction of the default animation.
+     */
+    spinDirection?: 'LEFT' | 'RIGHT';
+    /**
+     * Label position in relation to the SVG.
+     */
+    labelPosition?: 'BOTTOM' | 'RIGHT';
+    /**
+     * The text that shows when loading.
+     */
+    label?: string;
```

Comment:
Can we make this a a `string | ReactElement` to be in line with our other components label flexibility.

Reply:
Sure thing.

Reply:
Done.